authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/services/CaasUpdateService.kt

```diff
@@ -31,16 +32,19 @@ class CaasUpdateService(
     val connectorService: ConnectorService,
     val caasClient: CaasClient,
     val dapsClientService: DapsClientService,
+    @ConfigProperty(name = "quarkus.oidc-client.sovity.client-enabled") val isCaasClientEnabled: Boolean
```

Comment:
Tbh this violates Kotlin [coding convention about annotations](https://kotlinlang.org/docs/coding-conventions.html#annotations). I'm not going to die on this hill, but I would use the common injection here